chatwoot/chatwoot
# feat: Show banner message if Chatwoot update available

Fixes https://github.com/chatwoot/product/issues/216

#### How Has This Been Tested?

- Open the file `app/javascript/dashboard/App.vue`
- Update the line number 110
```
this.latestChatwootVersion = '5.0.1';
```

- Reload the dashboard

- Check the update banner are displaying on top of the page

![Screenshot 2022-02-28 at 3 40 41 PM](https://user-images.githubusercontent.com/12408980/155965499-44b2b863-b9b8-4493-9459-fcda135453cb.png)


- When click on dismiss icon, the banner should disappear
